docs/en_US/CommunitySharings/AutoCompletion.md

```diff
@@ -0,0 +1,39 @@
+# Auto Completion for nnictl Commands
```

Comment:
please mention it after installation doc to let users know there is auto completion support. for example, after QuickStart installation

Reply:
Sure, has added.